src/antsibull_changelog/config.py

```diff
@@ -381,6 +381,7 @@ class ChangelogConfig:
     add_plugin_period: bool
     changelog_nice_yaml: bool
     changelog_sort: str
+    vcs: str
```

Comment:
Can this be typed using `Literal` instead or does that cause other problems?

Reply:
f234432302a7277fc2c39f79db5ab23fe0f687bd